### PR TITLE
SS-8107 Implementing handling of "context" query string parameter

### PR DESCRIPTION
**Pull request by snabela**
https://shapediver.atlassian.net/browse/SS-8107

### CODE REVIEW

File: store/useShapeDiverStoreViewer.ts [hunk 1]
```diff
@@ -111,7 +112,24 @@ export const useShapeDiverStoreViewer = create<IShapeDiverStoreViewer>()(devtool
...
+			}
+			catch (e: any) {
+				if (isViewerGeometryBackendResponseError(e) && 
+					e.geometryBackendErrorType === ShapeDiverResponseErrorType.REQUEST_VALIDATION_ERROR &&
+					e.message.startsWith("Invalid parameter") && e.message.includes("'context'")) 
+				{
+					console.warn("Session creation failed due to invalid or missing 'context' parameter. Retrying without 'context' parameter.");
+
+					const dtoWithoutContext: SessionCreateDto = {...dto, initialParameterValues: {...dto.initialParameterValues}};
+					delete dtoWithoutContext.initialParameterValues!["context"];
```
Comment (MajorMeerkatThe3rd) [inline store/useShapeDiverStoreViewer.ts:126]: @snabela I'm not fully getting this line. If you assign the dto.initialParameterValues in the line before that, why is there a need to remove the context here? It shouldn't be there in this object anyway, right?

**Review by MajorMeerkatThe3rd — COMMENTED** [review]
@snabela I left a comment on a line that is not completely clear to me. Besides that all good!

File: store/useShapeDiverStoreViewer.ts (hunk 1 above)
Comment (snabela) [reply store/useShapeDiverStoreViewer.ts:126]: @MajorMeerkatThe3rd the line above just deep copies `dto.initialParameterValues`, but it doesn't remove `context`. Please note that `context` is already part of the `dto` object when this function is called. Does this answer your concern?

Comment (MajorMeerkatThe3rd) [reply store/useShapeDiverStoreViewer.ts:126]: @snabela Ah got it, I thought that was one level above that

**Review by MajorMeerkatThe3rd — APPROVED** [review]
@snabela All good, ready for merge 🚀

**Pull request merged**